Prepare for rebrand and domain switch

mattbatman opened the pull request:
For the switch from CheddarGetter to Cheddar and https://cheddargetter.com to https://getcheddar.com, this pull request:

- swaps our logo
- changes the configuration variables where appropriate
- where needed, manual changes were made to shorten "cheddargetter" to "cheddar" in articles and repositories we control

This pull request did not change links to open-source wrappers other than our PHP wrapper.

Commit:
Prepare for rebrand and domain switch

-swapped logo
-changed config variables
-manually change references where needed in certain links to articles
or sites we control

marcguyer commented on source/includes/_request-dictionary.md.erb:
```diff
@@ -376,7 +376,7 @@ Name | Description
 
 <p>
-  Check out this article for more information about <a href="<%= config[:support_url] %>/kb/getting-started-19/can-i-migrate-or-import-existing-customers-to-cheddargetter-how">
+  Check out this article for more information about <a href="<%= config[:support_url] %>/kb/getting-started-19/can-i-migrate-or-import-existing-customers-to-cheddar-how">
```
@mattbatman Need to make this and others conform to current KB article permalinks in Tender.

marcguyer replied:
To be clear, these need to be cross-referenced with what currently exists and make a change only as necessary. For example, this one was actually already changed back in the day to:

http://support.cheddargetter.com/kb/getting-started-19/customer-data-migration-and-import

marcguyer commented on source/includes/_wrappers.md.erb:
```diff
@@ -22,7 +22,7 @@ supported by the <%= config[:general_name] %> team.  There's a nice
 This ZF-compatible library is provided as an open source repository via GitHub
 here:
 
-* [http://github.com/marcguyer/cheddargetter-client-php](http://github.com/marcguyer/cheddargetter-client-php)
```
This should stay as cheddargetter for now.

marcguyer commented on source/index.html.md.erb:
```diff
@@ -1,5 +1,5 @@
 ---
-title: CheddarGetter API Reference
+title: Cheddar API Reference
```
Can this use the config? Maybe not since it's frontmatter?

mattbatman replied:
Right. There's a [closed Github issue at Middleman](https://github.com/middleman/middleman/issues/1246) that touches on this. I could change anything that uses the title front matter variable to instead rely on the global config variable. However, I suspect that doing so could increase the number of merge conflicts the next time we upgrade Slate.

marcguyer replied:
No big deal. Just leave it.

marcguyer commented on source/layouts/layout.erb:
```diff
@@ -73,7 +73,7 @@ under the License.
     <div class="page-wrapper">
       <div class="dark-box">
         <div class="ribbon right green">
-          <a href="https://github.com/sproutbox/cheddargetter-api-docs">Fork me on GitHub</a>
```
This should stay as cheddargetter for now but the link is actually this since the Github org was changed to `chdr`:

https://github.com/chdr/cheddargetter-api-docs

marcguyer commented on source/stylesheets/_variables.scss:
```diff
@@ -20,7 +20,7 @@ under the License.
 // Use these settings to help adjust the appearance of Slate
 
-@import 'http://cheddargetter.com/app/fonts/TradeGothic.css';
+@import 'http://getcheddar.com/app/fonts/TradeGothic.css';
```
I think we might have some cross-domain issues with this. There will likely be a server-side config change necessary. Research required regarding loading fonts across domains.

marcguyer replied:
Actually I don't think this will be a problem. We're already (somehow) adding the `Access-Control-Allow-Origin:*` header when serving this file. If it's a problem, we can fix it later.

Commit:
Changed links per requested changes from pull request #2

- API docs github repo under chdr
- don’t change PHP wrapper
- reviewed knowledge base links for most current version of permalink

marcguyer committed:
Merge branch 'master' into rebrandImplementation1071
